cmd/kubeadm/app/phases/copycerts/copycerts.go

```diff
@@ -89,7 +89,7 @@ func UploadCerts(client clientset.Interface, cfg *kubeadmapi.InitConfiguration,
 	fmt.Printf("[upload-certs] Storing the certificates in Secret %q in the %q Namespace\n", kubeadmconstants.KubeadmCertsSecret, metav1.NamespaceSystem)
 	decodedKey, err := hex.DecodeString(key)
 	if err != nil {
-		return err
+		return errors.Wrap(err, "error decoding certificate key")
```

Comment:
this change seems out of scope for this PR title?

Reply:
+1 for @neolit123 here

Reply:
Thanks for the review. I have made the necessary changes.